common/web/types/src/kmx/element-string.ts

```diff
@@ -24,14 +24,19 @@ export class ElemElement {
 ;
 
 export class ElementString extends Array<ElemElement> {
-  constructor(strs: Strs, source: string, order?: string, tertiary?: string, tertiary_base?: string, prebase?: string) {
+  constructor(strs: Strs, source: string | string[], order?: string, tertiary?: string, tertiary_base?: string, prebase?: string) {
     super();
     //TODO-LDML: full UnicodeSet and parsing
     if(!source) {
       return;
     }
 
-    let items = source.split("");
+    let items;
+    if (Array.isArray(source)) {
+      items = source;
+    } else {
+      items = source.split("");
+    }
```

Comment:
```suggestion
    let items = Array.isArray(source) ? source : source.split("");
```

Less verbose but just a nit grin